check_log_ng.py

```diff
@@ -354,16 +362,63 @@ def _check_each_single_line(
         fileobj.close()
         return end_position
 
+    def _create_digest_condition(self, logfile_pattern):
+        """Create the digest of search conditions."""
+        strings = []
+        for key in sorted(self.config):
```

Comment:
At strict thinking, "key" should append to `strings`.
Is this will cause serious problem?

Reply:
No, it will not.
I'll try it.